docs/docsite/rst/command_guide/index.rst

```diff
@@ -18,4 +18,11 @@ Ansible provides ad hoc commands and several utilities for performing various op
 
    intro_adhoc
    command_line_tools
-   cheatsheet
\ No newline at end of file
+   cheatsheet
+
+.. seealso::
+
+   `Ansible Navigator <https://ansible.readthedocs.io/projects/navigator/>`_
```

Comment:
This should probably be wired via intersphinx @ https://github.com/ansible/ansible-documentation/tree/devel/docs/docsite/sphinx_conf

Reply:
That should probably be a separate PR to add other intersphinx links to the conf.py

Reply:
No, if a reference to an external site for the first time must be coupled with adding that site. So that it's an atomic change.

Adding multiple unrelated entries to intersphinx would be a separate thing, though. That would require separate PRs and would need to replace corresponding references. Adding intersphinx entries without actually using them is not a good idea.

Reply:
I would still do adding an intersphinx link in conf.py and replacing this URL by a reference in a separate PR. Then it can be simply reverted in case of problems without removing this seealso.

Reply:
> I would still do adding an intersphinx link in conf.py and replacing this URL by a reference in a separate PR.

Why would one want to revert it separately, though? If it appears together, it should be reverted together, possibly with a partial revert, if needed.

Reply:
You want to be able to revert "adding the intersphinx link in conf.py and adding it in this place" together. But you want **not** to have to revert "adding the intersphinx link in conf.py and adding a seealso in some documentation" together.

Reply:
Arguably, there shouldn't be a revert but a normal change to a URL if it's decided to replace refs with direct URLs.